src/runtime/swg-client-event-manager.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2019 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as EventApi from '../api/swg-client-event-manager-api';
```

Comment:
We only usually allow `import *` in tests.

Reply:
Is there a good reason for that?  It really makes the code easier to read AND allows the intellisense to actually function.  If I have to take this out and I want to have something in the jsdoc that isn't actually used in code I need to refer to the file in the comment and then the intellisense / code complete doesn't work:

Ex:

  /**
   * Call this function to log an event.  The registered listeners will be
   * invoked unless the event is filtered.  Returns false if the event was
   * filtered and throws an error if the event is invalid.
   * @param {!../api/swg-client-event-manager-api.SwgClientEvent} event
   * @returns {!Promise}
   * @overrides
   */

In this case the event object is an "any" if I hover over it.  This makes it more difficult to code.

Reply:
Note that for this specific case since I've removed all the documentation it no longer matters, I'm just curious if there's a reason to avoid the syntax that seems cleaner to read and work with.

Reply:
Good question, especially in the context of implementation-of-an-interface scenario. In a generic, I think, there's some value for a reader in knowing early on which exports are used.

Reply:
I'd personally rather have code complete but I can see your argument